lib/babel.js

```diff
@@ -28,3 +28,9 @@ var transpiled = babel.transformFileSync(testPath, options);
 requireFromString(transpiled.code, testPath, {
 	appendPaths: module.paths
 });
+
+process.on('message', function (message) {
+	if (message['ava-kill-command']) {
+		process.exit(0);
```

Comment:
This means the test process is now receiving messages.

Do we need to monkey patch `process.on` so tests never see them?